lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -66,8 +66,6 @@ func NewImageCache() *ImageCache {
 
 // Update adds new images to the cache
 func (c *ImageCache) Update(client *client.PortLayer) error {
-	c.m.Lock()
-	defer c.m.Unlock()
 
 	CacheNotUpdated = true
```

Comment:
@caglar10ur does it make a difference that `Update()` is now only called one single time, at persona startup?